torch/csrc/utils/python_symnode.cpp

```diff
@@ -4,23 +4,53 @@ namespace torch {
 
 py::handle get_symint_class() {
   // NB: leak
+#if IS_PYBIND_2_13_PLUS
+  PYBIND11_CONSTINIT static py::gil_safe_call_once_and_store<py::object>
+      storage;
+  return storage
+      .call_once_and_store_result([]() -> py::object {
+        return py::module::import("torch").attr("SymInt");
+      })
+      .get_stored();
+#else
   static py::handle symint_class =
       py::object(py::module::import("torch").attr("SymInt")).release();
   return symint_class;
+#endif
```

Comment:
It feels like this code can benefit from good old `#define PYBIND_GET_TORCH_ATTR(name)` or a fancy template and this way avoid duplications